Sources/SwiftLayout/Extension/Anchors+Convenience.swift

```diff
@@ -123,9 +129,56 @@ extension Anchors {
     ///  - offset: CGFloat. top constant is `offset`, bottom constant is `-offset`, trailing constant is `-offset`
     ///
     /// - Returns: ``Constraint``
-    @AnchorsBuilder
-    public static func shoe(offset: CGFloat = .zero) -> SwiftLayout.Constraint {
-        horizontal(offset: offset)
-        Anchors(.bottom).equalTo(constant: -offset)
+    public static func shoe(offset: CGFloat = .zero) -> Anchors {
+        let horizontal = horizontal(offset: offset)
+        let bottom = Anchors(.bottom).equalTo(constant: -offset)
+        return horizontal.union(bottom)
+    }
+    
+    /// ``Anchors`` for width, height toward toItem: ``ConstraintableItem``
+    ///
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - length: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, length: CGFloat = .zero) -> Anchors {
+        size(toItem, size: .init(width: length, height: length))
+    }
+    
+    /// ``Anchors`` for width, height toward self
+    ///
+    /// - Parameters:
+    ///  - length: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size(length: CGFloat = .zero) -> Anchors {
+        size(.init(width: length, height: length))
+    }
+    
+    /// ``Anchors`` for CGSize toward toItem: ``ConstraintableItem``
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - size: constants
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, size: CGSize = .zero) -> Anchors {
```

Comment:
I think, "offset" will be better than "size" in this method

Reply:
this right too.